src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,132 @@
+.menu {
+  background-color: $green;
+
+  &__content {
+    box-sizing: border-box;
+    padding: 27px 20px;
+  }
+
+  &__list {
+    box-sizing: border-box;
+    list-style: none;
+    display: flex;
+    justify-content: space-evenly;
+
+    border-bottom: 1px solid #fff;
+    padding: 0 0 24px;
+    margin: 0;
+
+    @include onDesktop {
+      border-right: 1px solid #fff;
+      border-bottom: none;
+    }
+
+    @include onTablet {
+      border-right: 1px solid #fff;
+      border-bottom: none;
+    }
+  }
+
+  &__link {
+    width: max-content;
+
+    display: block;
+    font-family: "IBM Plex Sans", sans-serif;
```

Comment:
You should use the font-family defined in a global variable or mixin to keep consistency with other elements.